src/main/java/com/uid2/admin/vertx/service/ClientKeyService.java

```diff
@@ -217,10 +220,40 @@ private void handleClientByKeyId(RoutingContext rc) {
         }
     }
 
-    private LegacyClientKey getClientKey(String keyId) {
+    private LegacyClientKey getClientKeyByKeyId(String keyId) {
         return this.clientKeyProvider.getAll().stream().filter(legacyClientKey -> legacyClientKey.getKeyId().equals(keyId)).findFirst().orElse(null);
     }
 
+    private void handleClientByContact(RoutingContext rc) {
+        try {
+            String contact;
+
+            try {
```

Comment:
could we apply the same treatment here that we did in https://github.com/IABTechLab/uid2-admin/pull/205 to avoid the try-catch nesting? So something like:
```
String contact= rc.queryParam("contact").stream().findFirst().orElse(null);
if (contact== null) {
    ResponseUtil.error(rc, 400, "unable to parse contact");
    return;
}
```